.gitignore

```diff
@@ -26,6 +26,7 @@ vendor/*
 !vendor/tooltipster
 !vendor/tribe-select2
 !vendor/freemius
+!vendor/mt-a11y-dialog
```

Comment:
Also not sure why we do this, to include vendor packages into version control. Not sure where green team is with this, but would be nice to move away from doing this.

Reply:
I actually agree - and I believe we were doing it to keep it in the zip? I'll have to experiment and find a reasonable alternative.

Reply:
The normal usage here for vendor files was for composer packages we load up that we want to exclude from Git but still whitelist from the packager when it builds the zip file.

It looks like we are manually including the file into Git right now. I would rather see these files included in via Composer like we did with CE: https://github.com/moderntribe/events-community/blob/master/composer.json#L12

and ultimately exclude them from Git but keep them whitelisted for zip package handling.

Reply:
Done.